src/index.ts

```diff
@@ -64,6 +64,8 @@ export function eth2ValidatorPaths(validatorIndex: number): {
 } {
   return {
     withdrawal: `m/12381/3600/${validatorIndex}/0`,
+    // If this path is ever updated so that the last index is not 0, then must also update
+    // the signing derivation path below in deriveEth2ValidatorKeys
     signing: `m/12381/3600/${validatorIndex}/0/0`,
```

Comment:
Can we this path generation logic to a function and reuse, so never needed to worry later?

Reply:
I thought about that.  Started with a conditional and throwing an error for invalid case but both functions are right next to each other in the code, and its strictly spec driven and unlikely to change anytime soon,  so i figured a comment would be better (and faster cuz no code to run)